src/sentry/replays/blueprints/api.md

```diff
@@ -608,3 +608,107 @@ Retrieve a collection of click events associated with a replay.
     ]
   }
   ```
+
+## Replays Viewed [/organizations/<organization_slug>/replays-viewed/]
+
+- Parameters
+
+  - project (optional, string)
+  - id (required, array[string]) - The id of the replay to retrieve. Can be specified multiple times.
+  - per_page (optional, number) - The per page limit should match the number of ids requested.
```

Comment:
What's the purpose of this if it always matches the length of the ids array?

Reply:
There is no purpose.  I'll remove it.